### PR TITLE
Tidy via keyboard + Tidy in background

### PR DESCRIPTION
**Pull request by briangonzalez**
This PR makes it possible to Tidy all tabs using the keyboard shortcut:

<kbd>Alt</kbd>+<kbd>t</kbd> x2

This also opens up the possibility of Tidying using other methods like:

<kbd>Alt</kbd>+<kbd>t</kbd> <kbd>Alt</kbd>+<kbd>→</kbd> (Right)
<kbd>Alt</kbd>+<kbd>t</kbd> <kbd>Alt</kbd>+<kbd>←</kbd> (Left)

et cetera.

(Note: on OSX, Alt and Option are the same key)

### CODE REVIEW

**Comment by briangonzalez** [comment]
Fixes #65

**Comment by briangonzalez** [comment]
Fixes #64

File: app/scripts/content.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  const slug = `${ e.shiftKey ? '⇧-' : '' }${ e.ctrlKey ? '⌥-' : '' }${ e.metaKey ? '⌘-' : '' }${e.key}`
+  map[slug] = map[slug] || []
+  map[slug].forEach(clearTimeout)
+  map[slug].push(setTimeout(() => delete map[slug], 300))
+
+  const isDoubleKeypress =  map[slug].length === 2;
+  const isOptT = slug === '†';
+
+  if (isDoubleKeypress && isOptT) {
+    console.log(`Tidying via keypress!`)
```
Comment (iKlsR) [inline app/scripts/content.js]: Leftover @briangonzalez ?

**Comment by iKlsR** [comment]
@briangonzalez Nice to know the shortcuts but I don't see them documented in the app? 😨

File: app/scripts/content.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+const map = {}
+document.addEventListener('keydown', e => {
+  const slug = `${ e.shiftKey ? 'shift-' : '' }${ e.ctrlKey ? 'ctrl-' : '' }${ e.metaKey ? 'meta-' : '' }${e.key}`
+  map[slug] = map[slug] || []
+  map[slug].forEach(clearTimeout)
+  map[slug].push(setTimeout(() => delete map[slug], 300))
+  if (map[slug].length === 2 && slug === 'ctrl-c') {
+    console.log(`Tidying via `)
```
Comment (iKlsR) [inline app/scripts/content.js]: @briangonzalez More debug stuff?

Comment (briangonzalez) [inline app/scripts/content.js]: Yeah! But I think this got removed prior to merge :)

File: app/scripts/content.js (hunk 1 above)
Comment (briangonzalez) [inline app/scripts/content.js]: Yeah, good catch. I'll remove here shortly :)

**Pull request merged**